server/controllers/packing.js

```diff
@@ -14,7 +14,14 @@ export default {
   list: function(req, res) {
     Package.find().then(data => res.json(data))
   },
-
+  complete: async function(req, res) {
+    const singlePackageId = req.body.singlePackage
```

Comment:
I'd just call it `packageId`, if it was more than a single one it would be an array